docs/API.md

```diff
@@ -675,6 +676,81 @@ exports[`drinking flavors throws on octopus 1`] = `"yuck, octopus flavor"`;
 
 Check out [React Tree Snapshot Testing](http://facebook.github.io/jest/blog/2016/07/27/jest-14.html) for more information on snapshot testing.
 
+
+### Extending Jest Matchers
+
+Using descriptive matchers will help your tests be readable and maintainable. Jest has a simple API for adding your own matchers. Here is an example of adding a matcher:
+
+```js
+expect.extend({
+  toContainLetter(received, expected) {
```

Comment:
Agreed on the complication. Figured that would be a discussion point. Can simplify.

How would screenshots help? Are you thinking gifs?